### ISSUE TITLE
Notarization Using Script Fails to Execute

### ISSUE THREAD

**Issue by guiyanakuang** [hydraulic-software/conveyor#117]
**Describe the bug**
```
app {
  mac {
    notarization {
      script = "notarization.sh $ZIP"
    }
  }
```
The notarization.sh script is in the same directory as conveyor.conf

When running `conveyor make site`, the following error occurs:
❌ Task failed: Notarized Mac App for ARM: notarization.sh: Command not found (exit code 127)
env: notarization.sh: No such file or directory
Subprocess Failed: notarization.sh: Command not found (exit code 127)
env: notarization.sh: No such file or directory

**Expected behavior**
The script should be called and the notarization process should proceed normally.

**Additional context**
 - OS: MacOS 14.5 Intel
 - Version 14.3

**Comment by mikehearn** [comment]
Please check that `notarization.sh` is marked +x executable.

**Comment by mikehearn** [comment]
If that doesn't do the trick and you're a commercial user, please open a support ticket by emailing contact@hydraulic.dev and we'll work out why it can't find your script.

**Comment by guiyanakuang** [comment]
<img width="1345" alt="image" src="https://github.com/user-attachments/assets/4e956c86-7be2-4b48-a739-5249e0079a8d">

Sorry I am not a commercial user, I am going to open source my project and use conveyor

**Comment by mikehearn** [comment]
Double checking, it looks like it can be a bug. It may be looking for the command in the PATH or relative to where you run Conveyor from rather than the documented location next to the config file. Can you try changing the command to be `./notarization.sh` and running Conveyor from the directory with the config file in it?

**Comment by guiyanakuang** [comment]
You are correct, using `./notarization.sh` worked successfully. In fact, I had tried this before asking the question, but for some reason, it didn't work at that time. It's quite strange.
